hash/megadriv.xml

```diff
@@ -26771,7 +26813,7 @@ Black screen, writes in ROM area then jumps to PC=1a2000 (encryption?)
 	</software>
 
 	<software name="sonic3up" cloneof="sonic3">
-		<description>Sonic The Hedgehog 3 (USA, prototype 19931120)</description>
+		<description>Sonic The Hedgehog 3 (prototype, 19931120)</description>
```

Comment:
None of the other prototypes have a comma between the word “prototype” and the date.